include/bluetooth/direction.h

```diff
@@ -26,6 +26,87 @@ struct bt_df_adv_cte_tx_param {
 	uint8_t  *ant_ids;
 };
 
+enum BT_DF_CTE_RX_TYPE {
+	BT_DF_CTE_RX_TYPE_AOA = BIT(0),
+	BT_DF_CTE_RX_TYPE_AOD_1US = BIT(1),
+	BT_DF_CTE_RX_TYPE_AOD_2US = BIT(2),
+};
+
+#define BT_DF_CTE_RX_TYPE_ALL                                                  \
+	(BT_DF_CTE_RX_TYPE_AOA | BT_DF_CTE_RX_TYPE_AOD_1US |                   \
+	 BT_DF_CTE_RX_TYPE_AOD_2US)
+
+/* @brief Constant Tone Extension parameters for connectionless
+ * reception.
+ *
+ * @note cte_type is a bit field that provides information about type of CTE
+ * an application expects:
+ * - BT_DF_CTE_RX_TYPE_AOA for Angle of Arrival
+ * - BT_DF_CTE_RX_TYPE_AOD_1US for Angle of Departure with 1us switching slots
+ * - BT_DF_CTE_RX_TYPE_AOD_2US for Angle of Departure with 2us switching slots
+ *
+ * In case BT_DF_CTE_RX_TYPE_AOA is not set, members: slot_durations,
+ * num_ant_ids and ant_ids are not required and their values will be not verified
+ * for correctness.
+ */
+struct bt_df_per_adv_sync_cte_rx_param {
+	/* Bitmap with allowed CTE types: AoD 1us, AoD 2us, AoA */
+	uint8_t cte_type;
+	/** Antenna switching slots, one of possible values: 1us, 2us. */
+	uint8_t slot_durations;
+	/** Max number of CTEs to receive, 0 means receive continuously. */
+	uint8_t max_cte_count;
+	/** Length of antenna switch pattern */
+	uint8_t num_ant_ids;
+	/** Antenna switch pattern */
+	const uint8_t *ant_ids;
+};
+
+struct bt_df_per_adv_sync_iq_samples_report {
+	/** Channel index used to receive PDU with CTE that was sampled. */
+	uint8_t chan_idx;
+	/** The RSSI of the PDU with CTE (excluding CTE). */
+	int16_t rssi;
+	/** Id of antenna used to measure the RSSI. */
+	uint8_t rssi_ant_id;
+	/** Type of CTE:
```

Comment:
Would it make sense to add an enum for these values?
same for `slot_duration` and `packet_status`.

It would be better to have definitions in this file instead of the application using definitions in hci.h IMO.

Reply:
Yes I can create enums in direction.h for `cte_type`, `slot_duration` and `packet_status`. I would have done that already but I didn't want to duplicate the enums. Those in hci.h are used by controller, so they have to stay there.

What is your prefered way?